src/Websites/Websites.Test/ScenarioTests/WebAppTests.ps1

```diff
@@ -1159,6 +1160,17 @@ function Test-SetWebApp
 		Assert-AreEqual "" $webApp.SiteConfig.PhpVersion
 		Assert-AreEqual "1.2" $webApp.SiteConfig.MinTlsVersion
 
+		# set Custom Host Name(s)- Failed Scenario
+		$oldWebApp= Get-AzWebApp -ResourceGroupName $rgname -Name $webAppName
+		$CurrentWebApp = Set-AzWebApp -ResourceGroupName $rgname -Name $webAppName -HostNames $HN
+		
+		#Assert
+		$status
+		foreach($oldHN in $oldWebApp.HostNames)
+		{
+			Assert-True { $CurrentWebApp.HostNames -contains $oldHN }
+		}	
+
```

Comment:
Please find the above test case to handle script failure during adding the custom host names.